client/components/UI/Form/InputArray/index.tsx

```diff
@@ -44,7 +45,7 @@ interface IProps {
     errors: {[key: string]: any};
     showErrors: boolean;
     testId?: string;
-    getRef?(field: string, value: any): React.RefObject<any>;
+    getRef?(field: string, value: any): React.RefObject<CoverageEditorComponent>;
```

Comment:
input array is used in multiple places so ref won't always be `CoverageEditorComponent`?